tests/aws/services/apigateway/conftest.py

```diff
@@ -66,6 +68,10 @@
 }
 
 
+def is_next_gen_api():
+    return os.environ.get("PROVIDER_OVERRIDE_APIGATEWAY") == "next_gen" and not is_aws_cloud()
```

Comment:
nit: you can also use `config.APIGW_NEXT_GEN_PROVIDER` which does it already in config.py:

`APIGW_NEXT_GEN_PROVIDER = os.environ.get("PROVIDER_OVERRIDE_APIGATEWAY", "") == "next_gen"`